Dockerfile

```diff
@@ -50,9 +50,16 @@ RUN mkdir -p /usr/src/criu \
 
 # setup a playground for us to spawn containers in
 ENV ROOTFS /busybox
-RUN mkdir -p ${ROOTFS} \
-    && curl -o- -sSL 'https://github.com/docker-library/busybox/raw/a0558a9006ce0dd6f6ec5d56cfd3f32ebeeb815f/glibc/busybox.tar.xz' | tar xfJC - ${ROOTFS}
+RUN mkdir -p ${ROOTFS}
 
+RUN case $(uname -m) in \
```

Comment:
I would always recommend against using `uname` for platform detection, both because it will often return the wrong result (building `arm/v7` images on an `arm64` box, for example) and because for many ARM 32-bit platforms it gets [a little out of hand](https://github.com/tianon/gosu/issues/6#issuecomment-132239368), and would recommend instead using something like `dpkg --print-architecture` or `apk --print-arch` to get a simple and consistent string that represents the architecture of the image/userspace instead.

Reply:
@tianon 
Can you check it?
Thanks.